### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rlopezcoto**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -60,7 +60,7 @@ requirements:
     - astropy >=5.0,<6.dev0
     - bokeh >=2.0,<3.dev0
     - ctapipe >=0.19.2,<0.20.dev0
-    - ctapipe_io_lst >=0.22.3,<0.23.dev0
+    - ctapipe_io_lst~=0.23.0
```
Comment (maxnoe) [inline recipe/meta.yaml:63]: I dont think the ~= notation works in conda recipes... this is why it was using the two part version before.

Comment (maxnoe) [reply recipe/meta.yaml:63]: No, it doesn't: https://conda.io/projects/conda/en/latest/user-guide/concepts/pkg-specs.html#package-match-specifications

Comment (rlopezcoto) [reply recipe/meta.yaml:63]: we need the newest one, so:
`>=0.23.1,<0.24.dev0`
?

Comment (maxnoe) [reply recipe/meta.yaml:63]: It should be the same as the pypi package, which has what you put here: ` ~=0.23.0`, so the equivalent conda requirement is:

` >= 0.23.0,<0.24.dev0`

**Comment by maxnoe** [comment]
Follow up here: #11

**Pull request closed without merging**